sources/packages/api/src/database/database.module.ts

```diff
@@ -2,10 +2,12 @@ import { Module } from "@nestjs/common";
 import { TypeOrmModule } from "@nestjs/typeorm";
 import { LoggerModule } from "../logger/logger.module";
 import { DatabaseService } from "./database.service";
+import { ormConfig } from "../database/config/ormconfig";
 
-const config = require("../../ormconfig"); // eslint-disable-line
-
-const finalConfig: any = { ...config, schema: process.env.DB_SCHEMA || "sims" };
+const finalConfig: any = {
+  ...ormConfig,
+  schema: process.env.DB_SCHEMA || "sims",
```

Comment:
The schema is already set on `ormConfig`, can you please remove line 9 and directly use the `ormConfig` instead of the `finalConfig`?